# A toolbox to interactively create, customize and explore multi-systemic metabolic reaction networks: The MetaboNetworks toolbox, including MetaboGetworks.
A Matlab-based tool to create custom sub-networks using main substrate-product pairs as defined by the Kyoto Encyclopaedia of Genes and Genomes (KEGG). It can be used to explore transgenomic interactions, for example mammalian and bacterial associations. It calculates the shortest path between a set of metabolites (e.g. biomarkers from a metabonomic study) and plots the connectivity between metabolites as links in a network graph. The resulting graph can be changed and explored interactively. Furthermore, nodes in the graph are linked to the KEGG compound webpage.
A multi-systemic example data set is provided of a supra-organism, this includes reactions that can occur in compartments of homo sapiens, firmicutes and/or bacretoidetes species. The MetaboGetworks function can be used to generate custom multi-organism databases to use in the MetaboNetworks function.

[![DOI:10.1093/bioinformatics/btt612](http://img.shields.io/badge/DOI-10.1093/bioinformatics/btt612-0887BA.svg)](https://doi.org/10.1093/bioinformatics/btt612)

Run instructions:
> >> MetaboGetworks

> >> MetaboNetworks
